include/odp/api/spec/classification.h

```diff
@@ -640,10 +670,36 @@ void odp_cls_pmr_param_init(odp_pmr_param_t *param);
  *
  * @return Handle to the Packet Match Rule.
  * @retval ODP_PMR_INVALID on failure
+ *
+ * @note Matching PMR rule created through this function sets the CLS mark metadata
+ * of the packet to zero.
+ *
+ * @note Rules created through this function are equivalent to rules created through
+ * odp_cls_pmr_create() with the same PMR terms and with the additional option fields
```

Comment:
Should be odp_cls_pmr_create_opt().

I made the mistake in my comment and it ended up here. Sorry about that.

Reply:
Will be fixed in next version.